changelog-maker.js

```diff
@@ -51,6 +58,16 @@ function replace (s, m) {
   return s
 }
 
+function showUsage (removeMd) {
```

Comment:
How about we remove this argument since it's only ever going to come from `process.stdout.isTTY`